float64 distance
float64 cam_x
float64 cam_y
float64 cam_z
float64 probability
string bbox_class
bool detection_flag
